src/EventListener/DefaultLocaleListener.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentNegotiationBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+
+final class DefaultLocaleListener
```

Comment:
Discovered that the default locale isn't available when handling router errors, opened https://github.com/symfony/symfony/pull/29483.